starlite/openapi/controller.py

```diff
@@ -58,13 +59,40 @@ def get_schema_from_request(request: Request) -> "OpenAPI":
             An [OpenAPI][pydantic_openapi_schema.v3_1_0.open_api.OpenAPI] instance.
 
         Raises:
-            [ImproperlyConfiguredException][starlite.exceptions.ImproperlyConfiguredException]: If the application
-                `openapi_schema` attribute is `None`.
+            ImproperlyConfiguredException: If the application `openapi_config` attribute is `None`.
```

Comment:
Why this change? The reference docs wouldnt be generated like this. Unless there is a good reason for this, please change back.

Reply:
@Goldziher I think you should check out my previous review

Reply:
ok, as the reference tzar decides 😉